src/implementations/twig/components/menu/menu.html.twig

```diff
@@ -81,6 +81,12 @@
   data-ecl-menu
   data-ecl-menu-max-lines="{{ _max_lines }}"
   data-ecl-auto-init="Menu"
+  {% if _toggle is not empty %}
+    data-ecl-menu-label-open="{{ _toggle.link.label }}"
```

Comment:
this is dangerous, checking for the _toggle emptyness is not enough, _toggle.link could be undefined and it would throw an error

Reply:
i just wanted to prevent errors, but ok, like this we are not even adding the attribute in case it's not going to have a value, which i think it's better :)